src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IObjectCreationExpression.cs

```diff
@@ -5947,6 +5947,1015 @@ public void M(bool b)
 Block[B2] - Exit
     Predecessors: [B1]
     Statements (0)
+";
+            VerifyFlowGraphAndDiagnosticsForTest<BlockSyntax>(source, expectedFlowGraph, expectedDiagnostics);
+        }
+
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
+        [Fact]
+        public void ObjectCreationFlow_41()
```

Comment:
Follow up for https://github.com/dotnet/roslyn/pull/26389#discussion_r185965624.